### ISSUE TITLE
JSON vs Python objects

### ISSUE THREAD

**Issue by gdude2002** [prodicus/vocabulary#4]
Why do all the methods return a string of JSON? What advantage does this have over returning plain Python objects (dicts, lists, etc)?

It seems a little backwards to return JSON as part of the API that's supposed to be easy to use - now I also have to parse that myself. If you require JSON strings, perhaps you could provide a keyword argument, eg `json=True` or `json=False` that will handle this accordingly.

**Comment by prodicus** [comment]
Hey @gdude2002 , I understand your concern and I think there should be an option like `json=False` added.

Consider this to be added in the next release :+1:

**Comment by gdude2002** [comment]
:+1:

**Comment by prodicus** [comment]
So what if(that's a big if) someone would be using this in `javascript` and wanted to access the `list` being returned? Now I am not sure if you can inherently do that in `javascript` but I thought returning `JSON` objects would pretty much serve the purpose of delivering the data in a widely usable manner. Leave alone for `javascript`. 

Now I may be wrong on this, so it would be great if you could share your thoughts on this.

**Comment by gdude2002** [comment]
Well, sure, but you're writing a Python library here, not a JavaScript library. What use case do you have for this Python-bound data somehow ending up being used by a JavaScript program, and why can't that program request the data on its own?

**Comment by prodicus** [comment]
You have a point here @gdude2002 . So would returning a `list`  a good idea? Or a `dict`?

**Comment by gdude2002** [comment]
Whichever would be more appropriate for the data you're returning, yes. Most (valid) JSON data will translate directly to Python dicts, but there's no reason you can't clean them up a little before returning if you so wish.

**Comment by prodicus** [comment]
I feel that `list`'s would be good here (wordnet style). 

So instead of 

```python
>>> vb.meaning("levitate")
'[{"text": "(intransitive) Be suspended in the air, as if in defiance of gravity.", "seq": 0}, {"text": "(transitive) To cause to rise in the air and float, as if in defiance of gravity.", "seq": 1}]'
>>>
```

it would return 

```python
>>> vb.meaning("levitate")
["(intransitive) Be suspended in the air, as if in defiance of gravity.",  "(transitive) To cause to rise in the air and float, as if in defiance of gravity."]
>>>
```

when `json=False`

What do you think ?

**Comment by gdude2002** [comment]
Yeah, that makes sense to me.

**Comment by prodicus** [comment]
Closed by 7573f21. 

feel free to reopen it again if not the case @gdude2002
